### ISSUE TITLE
Updatelayers with --skip-unadvertised fails for cascaded WMS layers.

### ISSUE THREAD

**Issue by garnertb** [GeoNode/geonode#1889]
`updatelayers` throws an AttributeError when the `--skip-unadvertised` switch is used on a Cascaded WMS Layer.  This does not require any changes to GeoNode, however once https://github.com/boundlessgeo/gsconfig/pull/90 is merged, we need to bump the gsconfig dependency.

**Comment by jj0hns0n** [comment]
@garnertb any update on this from

**Comment by garnertb** [comment]
Hold.  Still pending boundlessgeo/gsconfig#90.

**Comment by jj0hns0n** [comment]
If I'm not mistaken, @ingenieroariel created a new gsconfig package after @ischneider pulled this PR. @garnertb can you test and close if possible?

**Comment by simod** [comment]
Any updates on this? Can we close it?

**Comment by capooti** [comment]
I wanted to test this but I am experiencing a different behavior of updatelayers from the past, so please forgive me If I now go off topic.
I created a cascaded layer from another GeoServer instance, and I can correctly access the layer in the GeoServer preview.
When running the updatelayers command now the layer is skipped (with 2.0 it would have been instead registered in GeoNode). Should I file another ticket with this, or this is the intended behavior?
